aioamqp/protocol.py

```diff
@@ -165,7 +165,12 @@ def start_connection(self, host, port, login, password, virtualhost, ssl=False,
         yield from self.open(virtualhost, capabilities='', insist=insist)
 
         # wait for open-ok
-        yield from self.dispatch_frame()
+        frame = yield from self.get_frame()
+        yield from self.dispatch_frame(frame)
+        if frame.frame_type == amqp_constants.TYPE_METHOD and \
+            frame.class_id == amqp_constants.CLASS_CONNECTION and \
```

Comment:
Perhaps use the more usual parenthesised and indented continuation, so the indentation of the block and the conditions differ.